jest.config.ut.js

```diff
@@ -0,0 +1,40 @@
+globalThis.ngJest = {
+  skipNgcc: true
+};
+
+/** @type {import('ts-jest/dist/types').JestConfigWithTsJest} */
+module.exports = {
```

Comment:
could be a factory to provide `moduleNameMapper` as well:
```typescript
/**
 * @params rootDir {string}
 * @returns {import(https://github.com/kulshekhar/ts-jest).JestConfigWithTsJest}
 */
module.exports.getJestConfig = (rootDir) => ({
   moduleNameMapper: getJestModuleNameMapper(rootDir)
  // ...
})
```

Reply:
much better